src/events/components/SearchModule/DateRangeInput.tsx

```diff
@@ -0,0 +1,65 @@
+import { DateTime } from 'luxon';
+import React, { FC, MutableRefObject, useRef } from 'react';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+
+export interface IProps {
+  dateStart: DateTime;
+  dateEnd: DateTime;
+  handleResetClick: () => void;
+  handleToDateClick: (day: Date) => void;
+  handleFromDateClick: (day: Date) => void;
+}
+
+export const DateRangeInput: FC<IProps> = ({
+  dateStart,
+  dateEnd,
+  handleToDateClick,
+  handleFromDateClick,
+  handleResetClick,
+}) => {
+  const modifiers = { start: dateStart.toJSDate(), end: dateEnd.toJSDate() };
+  const toInput = useRef(null) as MutableRefObject<null | DayPickerInput>;
+
+  return (
+    <div className="InputFromTo">
+      <button onClick={handleResetClick}>Rest</button>
+      <DayPickerInput
+        value={dateStart.toJSDate()}
+        placeholder="Fra"
+        format={'YYYY-M-D'}
+        dayPickerProps={{
```

Comment:
`dayPickerProps` makes up a large portion of the body here.

It could be moved outside of the JSX body at least?

something like this?:
``` typescript
const fromMonthProps: DayPickerProps = {
    selectedDays: [dateStart, { from: dateStart, to: dateEnd }],
    disabledDays: { after: dateEnd },
    toMonth: dateEnd,
    modifiers: { start: dateStart, end: dateEnd },
    numberOfMonths: 2,
    onDayClick: onDayClick,
};
```